examples/rds/provider.tf

```diff
@@ -0,0 +1,10 @@
+# Default Provider Configuration
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+# Aliased provider - used by Module resource
+provider "aws" {
+  alias  = "ap-southeast-2"
+  region = "ap-southeast-2"
+}
```

Comment:
Why are we setting 2 providers?

Reply:
I was unsure if you wanted to show how to use an alias or not - I believe there is no need for the alias provider, but know you use an alias. Maybe this is not the place to do that. I had added some documentation in the examples folder about this.

Reply:
Removed this..using a single provider after discussions